### DISCUSSION TITLE
How not to Specify Initial and Boundary Conditions.

### DISCUSSION THREAD

**Discussion by cdelv** [lululxvi/deepxde#813] [Q&A]
Many times we don't know the initial or boundary conditions. Fortunately, PINNs can work without them. To ensure the closure of the problem one can add some experimental data to the training.

I tried many ways to skip the definition of the Boundary and/or Initial conditions without success. Does someone know how to do this?

**Comment by lululxvi** [comment]
See https://deepxde.readthedocs.io/en/latest/modules/deepxde.data.html#deepxde.data.pde.PDE

"Use [] if no boundary condition."

**Reply by cdelv** [reply]
I see, thanks. In the number of points should I just put None?

**Reply by lululxvi** [reply]
Use 0
